ResourceStatusSystem/Client/ResourceStatusClient.py

```diff
@@ -52,12 +57,28 @@ def __init__( self ):
     self.rssService = RPCClient( "ResourceStatus/ResourceStatus" )
     self.validElements = RssConfiguration.getValidElements()
 
+
+  def _prepare(self, sendDict):
+
+    # remove unnecessary key generated by locals()
+    del sendDict['self']
+    del sendDict['element']
+    del sendDict['tableType']
+
+    # make each key name uppercase to match database column names (case sensitive)
+    for key, value in sendDict.items():
```

Comment:
sendDict.iteritems()?

Reply:
I can't as there's a `del sendDict[key]` just on the next line

Reply:
true